tests/ops/api/v1/endpoints/test_consent_request_endpoints.py

```diff
@@ -0,0 +1,304 @@
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Any
+from unittest.mock import patch
+
+import pytest
+
+from fidesops.ops.api.v1.urn_registry import (
+    CONSENT_REQUEST,
+    CONSENT_REQUEST_PREFERENCES,
+    CONSENT_REQUEST_VERIFY,
+    V1_URL_PREFIX,
+)
+from fidesops.ops.models.privacy_request import (
+    Consent,
+    ConsentRequest,
+    ProvidedIdentity,
+)
+
+
+@pytest.fixture
+def provided_identity_and_consent_request(db):
+    provided_identity_data = {
+        "privacy_request_id": None,
+        "field_name": "email",
+        "hashed_value": ProvidedIdentity.hash_value("test@email.com"),
+        "encrypted_value": {"value": "test@email.com"},
+    }
+    provided_identity = ProvidedIdentity.create(db, data=provided_identity_data)
+
+    consent_request_data = {
+        "provided_identity_id": provided_identity.id,
+    }
+    consent_request = ConsentRequest.create(db, data=consent_request_data)
+
+    yield provided_identity, consent_request
+
+
+@pytest.mark.usefixtures(
+    "email_config",
+    "email_connection_config",
+    "email_dataset_config",
+    "subject_identity_verification_required",
+)
+@patch("fidesops.ops.service._verification.dispatch_email")
+def test_consent_request(mock_dispatch_email, api_client):
+    data = {"email": "test@example.com"}
+    response = api_client.post(f"{V1_URL_PREFIX}{CONSENT_REQUEST}", json=data)
+    assert response.status_code == 200
+    assert response.json()["identity"]["email"] == data["email"]
+    assert mock_dispatch_email.called
```

Comment:
Looked a little closer at your tests -  you only have this one test for the CONSENT_REQUEST endpoint, I think this is where our existing pattern of class based organization around each endpoint is helpful https://github.com/ethyca/fidesops/pull/1387/files#r980411945, I think it helps more easily highlight to ourselves where we're lacking test coverage. Could you add more test coverage around the error cases here? 

Most of the actual bugs found in the first pass (not typos/postman collection issues) were on this particular endpoint.

Reply:
thanks so much for the added tests and organization here